src/ImageProcessorCore.Web/Middleware/ImageProcessorMiddleware.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.Extensions.PlatformAbstractions;
+using Microsoft.AspNet.Http;
+using System.IO;
+using System.Collections.Generic;
+using ImageProcessorCore.Samplers;
+using ImageProcessorCore.Filters;
+using System;
+using System.Globalization;
+
+namespace ImageProcessorCore.Web.Middleware
+{
+    public class ImageProcessorMiddleware
+    {
+        private readonly IApplicationEnvironment _appEnvironment;
+
+        private readonly RequestDelegate _next;
+
+        public ImageProcessorMiddleware(RequestDelegate next, IApplicationEnvironment appEnvironment)
+        {
+            _next = next;
+            _appEnvironment = appEnvironment;
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            if (!IsImageProcessorRequest(context))
+            {
+                // move to the next request here?
+                await _next.Invoke(context);
+                return;
+            }
+
+            int width = 0;
+            int.TryParse(context.Request.Query["width"], out width);
+            int height = 0;
+            int.TryParse(context.Request.Query["height"], out height);
+
+            var inputPath = _appEnvironment.ApplicationBasePath + "/wwwroot" + context.Request.Path;
```

Comment:
I'm assuming this was just to test that you could load images? In the Framework branch we have a concept of `IImageService` that is responsible for loading images from different locations, we should copy but also improve on that approach.